NEWS

```diff
@@ -1,9 +1,117 @@
 python-scrapinghub release notes
 ================================
 
+Version 1.9.0
+-------------
+(release on Nov 2nd, 2016)
```

Comment:
Note that starting with a digit seems to cause trouble with generated link (for Sphinx at least)
e.g for 2.12.3 (2016-12-01), the link is
`http://docs.python-requests.org/en/master/community/updates/#id1`

I "fixed" it in Scrapy by prepending "Scrapy " (https://github.com/scrapy/scrapy/pull/2294)